config/ipam/image_patch.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: ipam-controller-manager
-  namespace: capm3-system
+  namespace: ipam-provider-m3-system
```

Comment:
That is one weird name. Is there reasoning behind that? I expected to see simple `ipam-system` or similar

Reply:
I agree. Suggested `ipam-metal3-system` in the IPAM PR